src/ext/config.js

```diff
@@ -105,6 +106,7 @@ function PPConfigs() {
     cfConfig["spending-restrictions"]["status-code"] = [403];
     cfConfig["spend-action"]["redeem-method"] = "reload";
     cfConfig["issue-action"]["tokens-per-request"] = 30;
+    cfConfig["issue-action"]["request-identifiers"] = {"query-param": "__cf_chl_captcha_tk__", "body-param": "g-recaptcha-response", "post-processed": "captcha-bypass"};
```

Comment:
Some public documentation about these paths is available?

Reply:
There's no documentation for the query parameter. Not sure about the other ones.

Reply:
+1, added some more documentation